juniper_official/README.md

```diff
@@ -1 +1,16 @@
-Rules by juniper employees go here
+# Healthbot default KPIs
+#
+ 
+Contains readily consumable healthbot playbooks and rules which are specific to chassis, linecard, system, interface and protocols
+key performance indicators(KPIs). KPI rules collects the statistics from network devices then analyzes the data and act.
```

Comment:
Change the "KPI rules.." sentence to:
"KPI rules collect the statistics from network devices, then analyze the data and act."

Reply:
Corrected and committed.